CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.14)
+
+set(VERSION "0.4.1")
+project(cpuid LANGUAGES C CXX ASM VERSION ${VERSION})
+
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Would this request C++20 support and fail building on systems which don't have recent enough compiler? I'm not much versed with CMake, hence the question.
If that is indeed the case, we don't need such restrictions; C++11 and C99 are enough for libcpuid.

Reply:
Good point, will change rightaway

Reply:
Pushed.